services/minespace-web/src/components/Forms/incidents/IncidentForm.js

```diff
@@ -627,40 +627,38 @@ const renderUploadInitialNotificationDocuments = (
           {title}
         </Typography.Title>
       )}
-      {notInitialDocumentsInForm &&
-        (!formValues.status_code || formValues.status_code === "DFT") &&
-        !isFinalReviewStage && (
-          <>
-            <Col span={24}>
-              <Typography.Title level={4}>
-                Upload Supporting Notification Documentation
-              </Typography.Title>
-              <Typography.Paragraph>
-                Please upload any documents that support this written incident notification. You may
-                return later to upload additional documents as needed.
-              </Typography.Paragraph>
-            </Col>
-            <Col span={24}>
-              <Form.Item>
-                <Field
-                  id={INITIAL_INCIDENT_DOCUMENTS_FORM_FIELD}
-                  name={INITIAL_INCIDENT_DOCUMENTS_FORM_FIELD}
-                  onFileLoad={(document_name, document_manager_guid) =>
-                    handlers.onFileLoad(
-                      document_name,
-                      document_manager_guid,
-                      Strings.INCIDENT_DOCUMENT_TYPES.initial,
-                      INITIAL_INCIDENT_DOCUMENTS_FORM_FIELD
-                    )}
-                  onRemoveFile={parentHandlers?.deleteDocument}
-                  mineGuid={match.params?.mineGuid}
-                  component={IncidentFileUpload}
-                  labelIdle='<strong class="filepond--label-action">Supporting Document Upload</strong><div>Accepted filetypes: .kmz .doc .docx .xlsx .pdf</div>'
-                />
-              </Form.Item>
-            </Col>
-          </>
-        )}
+      {notInitialDocumentsInForm && !isFinalReviewStage && (
```

Comment:
I recommend turning on the "Hide whitespace differences" here!
This is mostly formatting- the only real changes were removing the condition `(!formValues.status_code || formValues.status_code === "DFT")` in 2 spots where we have `!isFinalReviewStage`. isFinalReviewStage is already false when status_code is undefined or DFT, so it was unnecessary to duplicate that logic.